common/static/js/es2015/ReactRenderer.jsx

```diff
@@ -0,0 +1,105 @@
+import 'babel-polyfill';
```

Comment:
Sorry I `git mv`'d this file but it doesn't look like it's showing the diff properly in this PR.